packages/lesswrong/components/ea-forum/EAHomeCommunityPosts.tsx

```diff
@@ -1,22 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Components, registerComponent } from '../../lib/vulcan-lib';
 import { Link } from '../../lib/reactRouterWrapper';
-import { AnalyticsContext } from '../../lib/analyticsEvents';
+import { AnalyticsContext, useTracking } from '../../lib/analyticsEvents';
 import moment from '../../lib/moment-timezone';
+import { useCookies } from 'react-cookie';
 import { useTimezone } from '../common/withTimezone';
 import { EA_FORUM_COMMUNITY_TOPIC_ID } from '../../lib/collections/tags/collection';
 
 const styles = (theme: ThemeType): JssStyles => ({
+  title: {
+    display: 'flex',
+    alignItems: 'center',
+    columnGap: 10
+  },
+  expandIcon: {
+    position: 'relative',
+    top: 2,
+    fontSize: 14,
+    cursor: 'pointer',
+    '&:hover': {
+      color: theme.palette.grey[800],
+    }
+  },
   readMoreLink: {
     fontSize: 14,
     color: theme.palette.grey[600],
-    fontWeight: 600
+    fontWeight: 600,
+    '@media (max-width: 350px)': {
```

Comment:
This requires no action in this PR, but it's something to consider in the future: I also added a media query recently with max-width 350px - maybe we should add this as another "xxs" breakpoint.